src/gentle_thrill_ride_type.cpp

```diff
@@ -387,8 +387,12 @@ void GentleThrillRideInstance::RecalculateRatings()
 	this->excitement_rating += scenery * t->excitement_increase_scenery;
 }
 
+static const uint32 CURRENT_VERSION = 1;   ///< Currently supported version of %GentleThrillRideInstance.
```

Comment:
and another one

I know they are local currently, but the same name for different things means you cannot move them together in one file if you wanted, also `CURRENT_VERSION` doesn't say what thing it gives a version number. They are all 1 now, but in time different `CURRENT_VERSION` will have different values, and then you have no simple way of checking if a `CURRENT_VERSION` variable is actually the correct one,.

Reply:
I stopped searching at this point, please do a global search to find all  of them.